etc/radar-jdbc-connector-data-dashboard-backend/values.yaml

```diff
@@ -0,0 +1,11 @@
+sink:
+  # Central topic for transformed topics (see docs/KSQLDB_transformer_for_data-dashboard-backend.md)
+  topics: ksql_observations
+  tableNameFormat: observation
+  # -- This config will create an auto-incrementing 'ID' column in the table.
+  insertMode: insert
+  mergeKey: false
+  incrementing.column.name: ID
+  primaryKeys:
+    mode: none
```

Comment:
Double check this is what you need

Reply:
Ah I reverted this commit. Indeed this was a leftover from the auto-increment column attempt.